HappyApp.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from time import sleep 
+from config import MessageType, Hour, day, _1PM, _8AM, _9PM
+from functions import isToday, scramble, alert
+from queue import Queue
+import readJSON
+from random import shuffle
+from mail import Mail
+
+class HappyApp:
+    def __init__(self,):
+        # For taking into account a sadir solider who was chosen to solo
+        self.sadirLeftOver = None
+        self.__roll()
+        self.mail = Mail()
+
+    def __roll(self):
+
+        # Read names
+        keva_names  = list(readJSON().keva().keys())
+        sadir_names = list(readJSON().sadir().keys())
+
+        # Remove from current round a sadir solider who soloed last round
+        if self.sadirLeftOver in sadir_names:
+            sadir_names.remove(self.sadirLeftOver)
+            self.sadirLeftOver = None
+
+        shuffle(sadir_names)
+
+        sadir_pairs = [(sadir_names[i],
+                       sadir_names[i+1])
+                       for i in range(0, len(sadir_names) - 1, 2)]
+
+        if len(sadir_names) % 2 != 0:
+            self.sadirLeftOver = sadir_names[-1]
+            sadir_pairs += sadir_names[-1]
+
+        tot_names = sadir_pairs + keva_names
+
+        shuffle(tot_names)
+        self.queue = Queue(tot_names)
```

Comment:
You can just `Queue(shuffle(sadir_pairs + keva_names)`

Reply:
Shuffle does not return a value. Can I still improve this?